dotCMS/src/main/java/com/dotmarketing/util/XMLUtils.java

```diff
@@ -60,18 +66,36 @@ public static boolean addPrologueIfNeeded(final File file)  {
 	
     public static String toXML10(final String input) {
         return xml10pattern.matcher(input).replaceAll("");
-        
-        
     }
     
     public static String toXML11(final String input) {
         return xml11pattern.matcher(input).replaceAll("");
-        
-        
     }
-    
-	
-	
+
+    /**
+     * Returns true if the XML is schemeless valid
+     * @param xml {@link String}
+     * @return boolean
+     */
+	public static boolean isValidXML (final String xml) {
+
+	    if (UtilMethods.isNotSet(xml)) {
+	        return false;
+        }
+
+        try {
+
+            final DocumentBuilder documentBuilder = factory.newDocumentBuilder();
+            documentBuilder.parse(new InputSource(new StringReader(xml)));
+        } catch (Exception e) {

```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5707914)